test/e2e/kamsuite/kamsuite.go

```diff
@@ -27,7 +33,7 @@ func FeatureContext(s *godog.Suite) {
 	s.Step(`^directory "([^"]*)" should exist$`,
 		DirectoryShouldExist)
 
-	s.Step(`^gitops repository is created$`,
+	s.Step(`^"([^"]*)" repository is created$`,
```

Comment:
I am not following what this step is.  May be you can add a comment.

Reply:
Previously we did not have the need to create any repository other than gitops. But for day2 operation, we are also using a bus repository. So we pass the name of the repository from [basic.feature](https://github.com/redhat-developer/kam/pull/282/files#diff-25040733ed2d476fece186ec77f656c9b53ddb09e77955c4d3e1b917901c4011R29) file and it'll create the repository with passed value.